### ISSUE TITLE
[Test Failure] Focus-rect color in themes for PM UI tabs

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#9336]
## Details about Problem
NuGet version: NuGet Client Dev\5.6.0.6529
VS Version: Master\29923.142
OS: Windows-10-Enterprise-RS5
## Detailed repro steps
1.	Create project in VS. 
2.	Open the solution level PM UI or project level PM UI.
3.	Tab through the PM UI and focus on the tabs. 
## Expected
The dash-line box should not be black when focusing on “Browse”/”Installed”/”Updates”/”Consolidate” tab in Dark theme。
## Actual
The dash-line box is black when focusing on “Browse”/”Installed”/”Updates”/”Consolidate” tab in Dark theme as below screenshot.
![image](https://user-images.githubusercontent.com/20632625/77430324-f3b02f80-6e15-11ea-8d67-d0002a9d219c.png)
## Note
It doesn't not repro in other themes.
